libraries/BLE/examples/notification/notification.ino

```diff
@@ -0,0 +1,81 @@
+
+#include "ArduinoBLE.h"
+#include "BLEAttribute.h"
+#include "BLECharacteristicImp.h"
+#include "BLEProfileManager.h"
+
+// LED pin
+#define LED_PIN   13
+
+// create service
+BLEService          ledService("19b10100e8f2537e4f6cd104768a1214");
+
+BLECharacteristic   switchCharacteristic("19b10101e8f2537e4f6cd104768a1214", BLERead | BLEWrite | BLENotify, 1);
+
+BLEDescriptor       switchDescriptor("2901", "switch");
+
+void setup() {
+    Serial.begin(115200);
+    Serial.println("test---");
+    
+    // set LED pin to output mode
+    pinMode(LED_PIN, OUTPUT);
+
+    // begin initialization
+    BLE.begin();
+    Serial.println(BLE.address());
+    
+    // set advertised local name and service UUID
+    BLE.setLocalName("LED");
+    BLE.setAdvertisedServiceUuid(ledService.uuid());
+    
+    // add service and characteristic
+    BLE.addService(ledService);
+    ledService.addCharacteristic(switchCharacteristic);
+    switchCharacteristic.addDescriptor(switchDescriptor);
+    unsigned char test = 1;
+    switchCharacteristic.writeValue(&test,1);

```

Comment:
> The typed characteristic can't be used, this is the only way. I have another question.

Why can't it be a typed characteristic? This is what we had in the v1 CurieBLE library ...

> If you want dismiss the writeValue for GATT server, how you resolve the write API for GATT client?
Because of this, I didn't implement the BLEAttributeWithValue class. I think writeValue and write is enough now.

Can you please expand on this, your question is not clear to me.